### PR TITLE
Capital snapshots

### LINKED ISSUE #34: capital line chart tile
(depot)

### PR DESCRIPTION
**Pull request by RohrerF**
Fixes #34 

- [x] Calculate the initial Values to make the Chart more beautiful

### CODE REVIEW

File: src/state/Config.ts [hunk 1]
```diff
@@ -10,13 +10,13 @@ interface StockMarketConfiguration {
 }
 
 export const StockConfig: StockMarketConfiguration = {
-    interval: 5,
-    lastMinutes: 60,
+    interval: 2,
```
Comment (panmona) [inline src/state/Config.ts]: how about 3?

**Review by panmona — DISMISSED** [review]

File: src/state/Config.ts (hunk 1 above)
Comment (RohrerF) [reply src/state/Config.ts]: do whatever you wan't. Idc

**Review by panmona — DISMISSED** [review]

File: src/state/depot/depotSaga.ts [hunk 2]
```diff
@@ -4,16 +4,16 @@ import { put, select, takeEvery } from 'redux-saga/effects';
...
 import { CapitalConfig as Config } from '../Config';
 import { INIT_SNAPSHOT_CAPITAL, setStockValueDevelopment, SNAPSHOT_CAPITAL, snapshotCapital } from './depotActions';
-import { getAccountValue, getCapital, getStockValueDevelopment } from './depotSelector';
+import { getCapital, getStockValueDevelopment } from './depotSelector';
 
 function* initSnapShots() {
 
     const values: FinancialSnapshot[] = [];
-    const capital = yield select( getAccountValue );
+    // const capital = yield select( getAccountValue );
```
Comment (panmona) [inline src/state/depot/depotSaga.ts]: remove?

File: src/state/Config.ts (hunk 1 above)
Comment (panmona) [reply src/state/Config.ts]: haha 🗡

**Review by panmona — APPROVED** [review]

**Pull request merged**